src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java

```diff
@@ -385,10 +381,10 @@ protected void additionalSetupForFileProcessing() throws IOException, ParseExcep
         fileNames.add(EMPTY_DATA_FILE);
         fileNames.add(GOOD_DATA_FILE_2);
 
-        BatchInputFileService batchInputFileService = EasyMock.createMock(CuBatchInputFileServiceImpl.class);
-        EasyMock.expect(batchInputFileService.listInputFileNamesWithDoneFile(EasyMock.isA(BatchInputFileType.class))).andReturn(fileNames);
-        EasyMock.expect(batchInputFileService.parse(EasyMock.isA(BatchInputFileType.class), EasyMock.isA(byte[].class))).andReturn(setupAchIncomeFilesGood()).andThrow(new RuntimeException()).andReturn(setupAchIncomeFilesGood());
-        EasyMock.replay(batchInputFileService);
+        BatchInputFileService batchInputFileService = mock(CuBatchInputFileServiceImpl.class);
+        when(batchInputFileService.listInputFileNamesWithDoneFile(isA(BatchInputFileType.class))).thenReturn(fileNames);
```

Comment:
I hadn't seen Mockito.isa before.  We have used Mockito.any in other cases.  Based on https://stackoverflow.com/questions/21430378/mockito-isa-any , we may want to use Mockito.any(BatchInputFileType.class) as the input for the function call.

Reply:
I'll look into this